cypress/e2e/tests/navbar.cy.js

```diff
@@ -0,0 +1,265 @@
+    cy.go(-1)
+    cy.location('pathname').should('not.include', 'docs')
+
+    // clicking forward
+    cy.go(1)
+    cy.location('pathname').should('include', 'docs')
+  })
+
+  it('cy.reload() - reload the page', () => {
+    cy.reload()
+
+    // reload the page without using the cache
+    cy.reload(true)
+  })
+
+  it('visits /docs/ directly', () => {
+    cy.visit('/docs/', {
+      timeout: 50000, // increase total time for the visit to resolve
+      onBeforeLoad (contentWindow) {
+        // contentWindow is the remote page's window object
+        expect(typeof contentWindow === 'object').to.be.true
+      },
+      onLoad (contentWindow) {
+        // contentWindow is the remote page's window object
+        expect(typeof contentWindow === 'object').to.be.true
+      },
+    })
+    cy.go(-1)
+    })
+})
+
+context('API Reference', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('ul.dropdown__menu > li')
+      .should('be.hidden')
+      .invoke('show')
+      .get('a.dropdown__link')
+      .contains('API Reference')
+      .click({force: true})
+  })
+
+  it('cy.go() - go back or forward in the browser\'s history', () => {
+
+    cy.location('pathname').should('include', 'apis')
+
+    cy.go('back')
+    cy.location('pathname').should('not.include', 'apis')
+
+    cy.go('forward')
+    cy.location('pathname').should('include', 'apis')
+
+    // clicking back
+    cy.go(-1)
+    cy.location('pathname').should('not.include', 'apis')
+
+    // clicking forward
+    cy.go(1)
+    cy.location('pathname').should('include', 'apis')
+  })
+
+  it('cy.reload() - reload the page', () => {
+    cy.reload()
+
+    // reload the page without using the cache
+    cy.reload(true)
+  })
+
+  it('visits /apis/ directly', () => {
+
+    cy.visit('/apis/', {
+      timeout: 50000, // increase total time for the visit to resolve
+      onBeforeLoad (contentWindow) {
+        // contentWindow is the remote page's window object
+        expect(typeof contentWindow === 'object').to.be.true
+      },
+      onLoad (contentWindow) {
+        // contentWindow is the remote page's window object
+        expect(typeof contentWindow === 'object').to.be.true
+      },
+    })
+    cy.go(-1)
+    })
+})
+
+context('SDKs', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('ul.dropdown__menu > li')
+      .should('be.hidden')
+      .invoke('show')
+      .get('a.dropdown__link')
+      .contains('SDKs')
+      .click({force: true})
+  })
+
+  it('cy.go() - go back or forward in the browser\'s history', () => {
+
+    cy.location('pathname').should('include', 'sdks')
+
+    cy.go('back')
+    cy.location('pathname').should('not.include', 'sdks')
+
+    cy.go('forward')
+    cy.location('pathname').should('include', 'sdks')
+
+    // clicking back
+    cy.go(-1)
+    cy.location('pathname').should('not.include', 'sdks')
+
+    // clicking forward
+    cy.go(1)
+    cy.location('pathname').should('include', 'sdks')
+  })
```

Comment:
**Food for Thought**: Do you see commonality between all forward/backward navigation Tests? Can it be abstracted?